src/goto-instrument/cover.h

```diff
@@ -46,18 +46,26 @@ struct cover_configt
 
 void instrument_cover_goals(
   const symbol_tablet &,
+  const cover_configt &,
   goto_functionst &,
   coverage_criteriont,
   message_handlert &message_handler);
 
 void instrument_cover_goals(
   const symbol_tablet &,
+  const cover_configt &,
   goto_programt &,
   coverage_criteriont,
   message_handlert &message_handler);
 
 std::unique_ptr<cover_configt> get_cover_config(
-  const optionst &, const symbol_tablet &, message_handlert &);
+  const optionst &,
+  const symbolt &,
+  const symbol_tablet &,
```

Comment:
I'd add a declarator to the `symbolt` parameter, to tell me what it does. All the others are obvious.

Reply:
Also, given that you are passing in the symbol_table, does it not make sense to give the identifier instead?